apis/api-directory/assets-api.md

```diff
@@ -0,0 +1,51 @@
+# Assets API
+
+{% swagger method="get" path="/assets/query" baseUrl="http://app.kubecost.com/query" summary="Assets API (Querying)" %}
+{% swagger-description %}
+The Assets API retrieves backing cost data broken down by individual assets in your cluster but also provides various aggregations of this data.
+{% endswagger-description %}
+
+{% swagger-parameter in="path" name="window" required="true" type="string" %}
+Duration of time over which to query. Accepts multiple formats including units of time, relative time units, and unix timestamps. See this section on [using the `window` parameter](/apis/api-directory/api-directory.md#using-the-window-parameter) for more information.
+{% endswagger-parameter %}
+
+{% swagger-parameter in="path" name="format" type="string" required="false" %}
+File type when exporting query. Currently only supports `json`.
+{% endswagger-parameter %}
+
+{% swagger-parameter in="path" name="aggregate" type="string" required="false" %}
+Used to consolidate cost model data. Supported values are `account`, `cluster`, `project`, `providerid`, `provider`, and `type`. Passing an empty value for this parameter or none at all returns data by an individual asset. Supports multi-aggregation (aggregation of multiple categories) in a comma separated list, such as `aggregate=account,project`.

```

Comment:
We actually support all these aggregation parameters, _and_ the `label:<name>` format
![image](https://github.com/kubecost/docs-cloud/assets/22826352/e69cc5fa-c345-485c-91e6-21c5d931c1f7)